partcad/src/partcad/shape.py

```
@@ -135,57 +209,51 @@ def do_change(self, change=None):
+                        pc_logging.info('Visualizing in "OCP CAD Viewer"...')
+                        # pc_logging.debug(self.shape)
+                        ocp_vscode.show(
+                            *components,
+                            progress=None,
+                            # TODO(clairbee): make showing (and the connection
+                            # to ocp_vscode) a part of the context, and memorize
+                            # which part was displayed last. Keep the camera
+                            # if the part has not changed.
+                            # reset_camera=ocp_vscode.Camera.KEEP,
+                        )
+                    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)